ocis-pkg/version/version.go

```diff
@@ -13,6 +13,12 @@ var (
 
 	// Date indicates the build date.
 	Date = time.Now().Format("20060102")
+
+	// Legacy defines the old long 4 number ownCloud version needed for some clients
+	Legacy = "10.11.0.0"
```

Comment:
Is there a better name for this?
It is also a string, so `Legacy` and `LegacyString` are not obvious when reading the code, to know if it is the 4-number or 3-number version string.